src/configurations/destinations/iterable/schema.json

```diff
@@ -158,7 +163,9 @@
         "type": "object",
         "properties": {
           "web": {
-            "type": "string"
+            "type": "string",
+            "enum": ["top-right", "top-left"],
```

Comment:
why this change is here?

Reply:
ideally this should be part of a seperate fix PR, but it is discussed, you can go ahead.